blocks/header/header.css

```diff
@@ -613,6 +616,13 @@ header a.button:hover span.button-text {
     margin: 0 12px;
   }
 
+  .mega-menu div h2 > a {
+    padding-left: 20px;
```

Comment:
Could you also apply the same hover effect as in the original page:

Ours:
<img width="328" alt="Screenshot 2023-09-01 at 10 43 55" src="https://github.com/hlxsites/bitdefender/assets/30388999/7b43e950-4328-4665-b128-263a534374de">

Original:
<img width="328" alt="Screenshot 2023-09-01 at 10 44 24" src="https://github.com/hlxsites/bitdefender/assets/30388999/18441903-158d-4d6b-9521-c323583c0885">

Reply:
Good catch.